torch/distributed/fsdp/_fully_shard/_fully_shard.py

```diff
@@ -46,6 +48,7 @@ def fully_shard(
     shard_placement_fn: Optional[Callable[[nn.Parameter], Optional[Shard]]] = None,
     mp_policy: MixedPrecisionPolicy = MixedPrecisionPolicy(),
     offload_policy: OffloadPolicy = OffloadPolicy(),
+    ignored_params: set[nn.Parameter] = set(),
```

Comment:
nit: I think it is a good practice to avoid mutable default args (https://docs.python-guide.org/writing/gotchas/) in case the function mutates it (which might not be the case here).
```suggestion
    ignored_params: Optional[set[nn.Parameter]] = None,
```

```
if ignored_params is None:
    ignored_params = set()
```